components/footer/footermenu.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from 'react'
+import { PageType } from '@/components/Types'
+
+export default function FooterMenu(
+  { data }: PageType,
+  { locale }: { locale: string },
+) {
+  const menu = 'תפריט'
```

Comment:
Next time if you have missing strings let me know instead of adding constant text like this.

Reply:
I sent a message in asana